src/app/forms/custom-sections/body/body.component.ts

```diff
@@ -79,9 +73,7 @@ export class BodyComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   get numberOptions(): MultiOptions {
-    return Array
-      .from(Array(10).keys())
-      .map(i => ({ value: i, label: `${i}` }));
+    return Array.from(Array(10).keys()).map(i => ({ value: i, label: `${i}` }));
```

Comment:
Not that it's super important, but I was staring at this function and thought if there was a way of  using `Object.fromEntries`. Turns out there isn't but this is the only improvement could come up with. `Array.from` can take a mapping function. So yeah just a suggestion
```suggestion
    return Array.from(Array(10).keys(), i => ({ value: i, label: `${i}` }));
```

Reply:
I've not actually changed this file, just my linter ran, so I'm gunna leave as is, sorry!